src/main/kotlin/de/bund/digitalservice/useid/identification/IdentificationSessionsController.kt

```diff
@@ -1,26 +1,33 @@
 package de.bund.digitalservice.useid.identification
 
+import de.bos_bremen.gov.autent.common.Utils
+import de.governikus.autent.sdk.eidservice.tctoken.TCTokenType
 import mu.KotlinLogging
 import org.springframework.http.HttpStatus
 import org.springframework.http.MediaType
 import org.springframework.http.ResponseEntity
+import org.springframework.http.server.reactive.ServerHttpRequest
 import org.springframework.web.bind.annotation.ExceptionHandler
 import org.springframework.web.bind.annotation.GetMapping
 import org.springframework.web.bind.annotation.PathVariable
 import org.springframework.web.bind.annotation.PostMapping
 import org.springframework.web.bind.annotation.RequestBody
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RestController
+import org.springframework.web.util.UriComponentsBuilder
 import reactor.core.publisher.Mono
+import reactor.core.scheduler.Schedulers
+import java.net.URLEncoder
 import java.util.UUID
 
 internal const val IDENTIFICATION_SESSIONS_ENDPOINT = "/api/v1/identification/sessions"
+internal const val TCTOKEN_ENDPOINT = "tc-token"
```

Comment:
Maybe we should rename the above as "prefix"? Otherwise, this looks a bit weird here and you could think that the endpoints are `/api/v1/identification/sessions` and `/tc-token`, even though the latter is `/api/v1/identification/sessions/tc-token".

Reply:
```kotlin
internal const val PREFIX_IDENTIFICATION_ENDPOINT = "/api/v1/identification"
internal const val SESSIONS_ENDPOINT = "$PREFIX_IDENTIFICATION/sessions"
internal const val TCTOKEN_ENDPOINT = "tc-token"
```
or 
```kotlin
internal const val PREFIX= "/api/v1/identification/sessions"
internal const val TCTOKEN_ENDPOINT = "tc-token"
```

Reply:
I would suggest: 
```
internal const val IDENTIFICATION_SESSIONS_BASE_PATH = "/api/v1/identification/sessions"
internal const val TCTOKEN_PATH_SUFFIX = "tc-token"
```